test/transport.test.js

```diff
@@ -18,7 +18,10 @@ test('pino.transport with file', async ({ same, teardown }) => {
     os.tmpdir(),
     '_' + Math.random().toString(36).substr(2, 9)
   )
-  const transport = pino.transport(join(__dirname, 'fixtures', 'to-file-transport.js'), { destination })
+  const transport = pino.transport({
+    src: join(__dirname, 'fixtures', 'to-file-transport.js'),
+    opts: { destination }
```

Comment:
would be cleaner to spell out `options` for public API, same for `source`.

altho `source` would be too abstract, perhaps a more specific name here.

Reply:
I have been at a loss for good names for these options.. suggestions are welcomed.

@jsumners @davidmarkclements wdyt about using long names?

Reply:
I am fine with using full names.

Reply:
Long names it is!